### ISSUE TITLE
How does this PPO implementation update exactly?

### ISSUE THREAD

**Issue by ghost** [OctThe16th/PPO-Keras#9]
Hi LuEE-C,

When i tried to log the output, Softmax i came around this:

![2019-11-18_16h41_38](https://user-images.githubusercontent.com/36861945/69069063-1f657800-0a26-11ea-87b8-f9d97003501e.png)

This was done with an ER size of 2500. It is because it only updates the network every full ER? But as you may see, this causes it to be a bit weird looking? Also is that even a thing to update it only every ersize steps, 2500 in this case?

Thank you for your response in advance,

Jan
